apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/SpringConfiguration.java

```diff
@@ -460,4 +463,16 @@ public HealthCheckRegistry healthCheckRegistry() {
     HealthCheckRegistry healthCheckRegistry = new HealthCheckRegistry();
     return healthCheckRegistry;
   }
+
+  /**
+   * @param includeFakeDrugCodeText the {@link String} to use
+   * @return the {@link FDADrugUtils} for the application, uses the current prod implementation or
+   *     the fake drug code for test.
+   */
+  @Bean
+  public FDADrugUtils drugCodeProvider(
+      @Value("${" + PROP_INCLUDE_FAKE_DRUG_CODE + ":false}") String includeFakeDrugCodeText) {
```

Comment:
a little bit of searching tells me that we can probably have a boolean property instead of a string property with conversion. can you try to get that to work?